test/blackbox-tests/test-cases/menhir/explain-env.t/run.t

```diff
@@ -0,0 +1,10 @@
+  $ dune build a/test.exe --debug-dependency-path
+  Warning: one state has shift/reduce conflicts.
+  Warning: one shift/reduce conflict was arbitrarily resolved.
+  $ ls _build/default/a/parser.conflicts
+  _build/default/a/parser.conflicts
```

Comment:
Could you do `dune build _build/default/a/parser.conflicts` directly so that we can see Dune recognizes it as a target and isn't producing it by accident.

Reply:
The reason I ask is that if Dune isn't actually aware of the .conflicts file in this situation then it cannot be depended on, or expected to persist between builds.